src/API/useCharacters.ts

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from "react";
+
+interface Result {
```

Comment:
Maybe a better name here can be used, like CharachtersFetchResult

Reply:
yes, you're right, the name was too general, I changed it